pom.xml

```diff
@@ -10,6 +10,7 @@
         <module>data</module>
         <module>management</module>
         <module>account</module>
+        <module>samples</module>
```

Comment:
the fact that samples are not part of the POM project is voluntary: It's just to avoid to publish them to maven central (We could probably edit some settings in the release/deploy script)